coforma/swift-tech-challenge
# Issue/Front end framework

# Issue/Front end framework

# Summary

The frontend framework to be used is NextJS, as documented in the associated ADR. This pull request sets up NextJS, including local development and helpful tooling. It contains ESLint and Prettier configuration, along with pre-commit checks for both, in addition to secret checking. It also configures a minimal baseline of USWDS styling and the USWDS React component library.





Implements issue #4 

## Type of change

## Major Changes
This will require developers to re-run `yarn install` and `pre-commit install` at the package root.

# How To Test
1. Follow directions for getting started with the repo (including installation of nvm, node, yarn, and pre-commit.
2. `cd` to the project directory and run `pre-commit install` to install pre-commit git hook scripts. This is what will guarantee linting, formatting, and secret-checking occur before each commit.
3. run `yarn dev`, then navigate to `localhost:3000` to verify the app is running successfully.

# Checklist:
